src/lumigo_tracer/wrappers/aiohttp/aiohttp_wrapper.py

```diff
@@ -0,0 +1,87 @@
+from lumigo_tracer.lumigo_utils import lumigo_safe_execute, get_logger, concat_old_body_to_new
+from lumigo_tracer.libs.wrapt import wrap_function_wrapper
+from lumigo_tracer.spans_container import SpansContainer
+from lumigo_tracer.wrappers.http.http_data_classes import HttpRequest
+from lumigo_tracer.wrappers.http.sync_http_wrappers import add_request_event, update_event_response
+
+try:
+    import aiohttp
+except Exception:
+    aiohttp = None
+
+LUMIGO_SPAN_ID_KEY = "_lumigo_span_id"
+trace_config = None
+
+
+def aiohttp_session_init_wrapper(func, instance, args, kwargs):
+    with lumigo_safe_execute("aiohttp aiohttp_session_init_wrapper"):
+        traces = kwargs.get("trace_configs") or []
+        traces.append(trace_config)
+        kwargs.update({"trace_configs": traces})
+    return func(*args, **kwargs)
+
+
+async def on_request_start(session, trace_config_ctx, params):
+    with lumigo_safe_execute("aiohttp on_request_start"):
+        span = add_request_event(
+            span_id=None,
+            parse_params=HttpRequest(
+                host=params.url.host,
+                method=params.method,
+                uri=str(params.url),
+                headers=dict(params.headers),
```

Comment:
What can it be if not a `dict`? Is it always something that can be converted to a `dict`?

Reply:
This is `CIMultiDict` from the docs, can't be something else. If it will be - I have protection :)